### DISCUSSION TITLE
Is there plans to integrate Dioxus and Tauri ?

### DISCUSSION THREAD

**Discussion by tkr-sh** [tauri-apps/tauri#9169] [General]
Is there plans to integrate Dioxus and Tauri ?
Dioxus is a multi-platform framework for building front-end applications in Rust.
Currently, yew, leptos and sycamore are supported but not Dioxus.
Is there a reason for that ?

**Comment by amrbashir** [comment]
Dioxus has its own desktop framework based on `wry` and `tao` crates so didn't feel like it needs its own option in create-tauri-app.

That said, you can still use Dioxus with tauri if you want to use it as a frontend framework only, you can configure the project manually, see https://tauri.app/v1/guides/getting-started/setup/integrate. Also there is a PR that is trying to add Dioxus here https://github.com/tauri-apps/create-tauri-app/pull/426/

**Reply by tkr-sh** [reply]
I mean, yes!
But the difference is that the ecosystem for mobile in Dioxus is at its start, and the main focus isn't much about mobile but making the Desktop experience more reliable.
In that sens, making it as a possible start option with like the other at the start, makes sens.
And tauri v2 being focused on mobile, it will make sens to use tauri w/ dioxus soonly

```sh
? Choose your UI template ›
❯ Vanilla
  Yew
  Leptos
  Sycamore
  ```
  
  Thanks for your return!

**Reply by amrbashir** [reply]
I see, I don't personally mind adding Dioxus as another option for Rust UI but I don't have the capacity to add it soon (also not a very good dioxus developer).
